app/src/androidTest/java/net/skyscanner/backpack/calendar/BpkCalendarTest.kt

```diff
@@ -78,7 +77,6 @@ class BpkCalendarTest : BpkSnapshotTest() {
   }
 
   @Test
-  @FlakyTest
```

Comment:
This was left over from before we started to use firebase, they are not flaky anymore.